pkgs/by-name/ra/rainbowcrack/package.nix

```
@@ -0,0 +1,49 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  alglib,
+  unzip,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rainbowcrack";
+  version = "1.8";
+
+  src = fetchurl {
```

```suggestion
  src = fetchzip {
```

It should unzip for you

---

fetchzip still having issue unpacking.